tests/baselines/reference/isolatedDeclarationErrorsExpandoFunctions.errors.txt

```diff
@@ -16,18 +21,33 @@ isolatedDeclarationErrorsExpandoFunctions.ts(8,1): error TS9023: Assigning prope
     foo.apply = () => {}
     ~~~~~~~~~
 !!! error TS9023: Assigning properties to functions without declaring them is not supported with --isolatedDeclarations. Add an explicit declaration for the properties assigned to this function.
+                ~~~~~~~~
+!!! error TS9007: Function must have an explicit return type annotation with --isolatedDeclarations.
+!!! related TS9030 isolatedDeclarationErrorsExpandoFunctions.ts:3:13: Add a return type to the function expression.
     foo.call = ()=> {}
     ~~~~~~~~
 !!! error TS9023: Assigning properties to functions without declaring them is not supported with --isolatedDeclarations. Add an explicit declaration for the properties assigned to this function.
+               ~~~~~~~
+!!! error TS9007: Function must have an explicit return type annotation with --isolatedDeclarations.
+!!! related TS9030 isolatedDeclarationErrorsExpandoFunctions.ts:4:12: Add a return type to the function expression.
     foo.bind = ()=> {}
     ~~~~~~~~
 !!! error TS9023: Assigning properties to functions without declaring them is not supported with --isolatedDeclarations. Add an explicit declaration for the properties assigned to this function.
+               ~~~~~~~

```

Comment:
Isn't this error redundant? If `foo.bind` is typed (somehow - interface merge maybe), you don't need types on this lambda.

Reply:
You are right. Removed it.